### ISSUE TITLE
[Interpreter] Float argument support for main

### ISSUE THREAD

**Issue by ayakayorihiro** [sampsyo/bril#159]
The current interpreter for Bril does not support `float` arguments to `main`.
Here's a simple piece of code:
```
@main (x1 : int, x2: float) {
      print x1;
      print x2;
}
```
The output of this is as follows:
```
$ bril2json < floats.bril | brili 2 3.0
2
error: undefined variable x2
```
x1 got printed successfully because it's an int, but it seems like the argument x2 results in an error because it's a float.

**Comment by sampsyo** [comment]
Thanks for reporting! Indeed; here's where those `main` arguments are handled:
https://github.com/sampsyo/bril/blob/b37c9ee2c0a446474ed71e82cf7889c042add53d/bril-ts/brili.ts#L803-L812

As you can see, it only supports `int` and `bool`. Two observations:

* We should add a `float` case using [`parseFloat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseFloat).
* The `switch` also lacks a `default` case; it would be better to report an error there indicating that the type is unsupported for CLI arguments.

**Comment by czarku7** [comment]
7

**Comment by sampsyo** [comment]
Fixed in #182.
